api/handlers/find_user.go

```diff
@@ -16,21 +15,28 @@ type findUser struct{
 	rt *ShivaniCustomServerExample1.Runtime
 }
 func (f *findUser) Handle(fup users.FindUsersParams) middleware.Responder{
-	// fup.Name
-	// v:=service.NewManager()
-	// v.ListUsers()
-	
-	fmt.Println("Application Name is",f.rt.GetApplicationName())
-	// n:="Shivani Sharma"
-	// us:= []*models.User{{Address:"ABC",ID:2,Name:&n}}
-	
-	us,_:= f.rt.GetManager().ListUsers(fup.Name,fup.Limit)
+
+	var limit int32
+	// var name string
+	var us []*domain.User
+	// if fup.Limit != nil && fup.Name != nil {
+		limit = *fup.Limit
+		// name = *fup.Name
+		us,_ = f.rt.GetManager().ListUsers(filteredMap(fup.Name), limit)
+	// }else{
+	// 	us,_ = f.rt.GetManager().ListUsers("", limit)
+	// }
 	usResponse:=[]*models.User{}
 	for _,usr:=range us{
 		usResponse=append(usResponse,asUserResponse(usr))
 	}
 	res:=users.NewFindUsersOK().WithPayload(usResponse)
 	return res
-	// print the appication name
 }
-
+func filteredMap(Name *string) map[string]interface{}{
```

Comment:
function name `extractFilters` will be more suitable.

Reply:
Done ! Please check